### PR TITLE
migrate from packr to embed

### PR DESCRIPTION
**Pull request by ramonskie**
as packr is currently deprecated and the many issues we had with creating  releases that caused strange issues with embing the terraform binary  we decided to switch it to go embed as this is since the release of go 1.16 a native solution to embed static files.

i paired together on this with @nouseforaname

### CODE REVIEW

File: terraform/binary_path.go [hunk 1]
```diff
@@ -1,22 +1,21 @@
...
 	"time"
 
-	"github.com/gobuffalo/packr/v2"
 	"github.com/spf13/afero"
 )
 
 const (
 	tfBinDataAssetName = "terraform"
 	bblTfBinaryName    = "bbl-terraform"
-	terraformBinary    = "./binary_dist"
```
Comment (rkoster) [inline terraform/binary_path.go:18]: Why was this removed?

File: bosh/executor_test.go [hunk 2]
```diff
@@ -89,7 +90,7 @@ var _ = Describe("Executor", func() {
 
 				contents, err = fs.ReadFile(nestedPath)
 				Expect(err).NotTo(HaveOccurred())
-				Expect(string(contents)).To(Equal("vsphere-cpi"))
+				Expect(string(contents)).To(Equal("aws-cpi"))
```
[the remark this answers is not in the document]
Comment (ramonskie) [reply bosh/executor_test.go:93]: because we wrote the content vsphere-cpi in an aws mocked file
`box.AddString("jumpbox-deployment/aws/cpi.yml", "vsphere-cpi")`

File: terraform/binary_path.go (hunk 1 above)
Comment (ramonskie) [reply terraform/binary_path.go:18]: this is removed as it was a constant that was used within the packr code block

**Review by rkoster — APPROVED** [review]

**Comment by rkoster** [comment]
@nouseforaname given that you already have context on this PR could you be so kind as to add a review?

**Review by nouseforaname — APPROVED** [review]
I think it's sane. We've tried to keep the changes to a minimum
